charts/kube-prometheus-stack/templates/prometheus-operator/admission-webhooks/validatingWebhookConfiguration.yaml

```diff
@@ -3,14 +3,14 @@ apiVersion: admissionregistration.k8s.io/v1
 kind: ValidatingWebhookConfiguration
 metadata:
   name:  {{ template "kube-prometheus-stack.fullname" . }}-admission
-{{- if .Values.prometheusOperator.admissionWebhooks.certManager.enabled }}
   annotations:
-    certmanager.k8s.io/inject-ca-from: {{ printf "%s/%s-admission" (include "kube-prometheus-stack.namespace" .) (include "kube-prometheus-stack.fullname" .) | quote }}
-    cert-manager.io/inject-ca-from: {{ printf "%s/%s-admission" (include "kube-prometheus-stack.namespace" .) (include "kube-prometheus-stack.fullname" .) | quote }}
-{{- end }}
+    {{- include "kube-prometheus-stack.prometheus-operator-webhook.annotations" $ | trim |nindent 4 }}
+    {{- with .Values.prometheusOperator.admissionWebhooks.validatingWebhookConfiguration.annotations }}
+    {{ toYaml . }}
```

Comment:
this sound suspicious to me. no indents?

Could you please add `prometheusOperator.admissionWebhooks.validatingWebhookConfiguration.annotations` to an existing ci test values file?

Reply:
done